.github/workflows/deploy-tag.yml

```
@@ -33,9 +33,8 @@ jobs:
     steps:
     - uses: actions/download-artifact@v1
       with:
-        name: built-extension
-        path: distribution
-    - run: cd distribution && npx chrome-webstore-upload-cli upload --auto-publish
+        name: distribution
```

This should be `path`

```suggestion
        path: distribution
```

---

No, path is automatically the same name as the artifact

https://github.com/actions/download-artifact#usage

---

When `download-artifact@v2` is available we can drop the name too:

https://github.com/actions/download-artifact/blob/3bdf740f024d67bba81315347e81ea6450d53b5b/src/download-artifact.ts#L11-L13